Assets/Scripts/Control&Input/ItemSelectManager.cs

```diff
@@ -1,19 +1,35 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using Mirror;
 using UnityEngine;
 
-public class ItemSelectManager : MonoBehaviour
+// TODO remove first two structs if unused
+internal struct StartTrackingMessage : NetworkMessage
+{
+}
+
+internal struct ReadyToTrackMessage : NetworkMessage
+{
+}
+
```

Comment:
Maybe remove the two above unless they are important for #661 ?

Reply:
Removing in #661 as these are not needed anymore